run_F2010SC5-P3_cori.csh

```diff
@@ -0,0 +1,1519 @@
+#! /bin/csh -fe
```

Comment:
Do we want to integrate this script as well or is this added by mistake?

Reply:
Hi @singhbalwinder, this is just a pull request to Koby's branch that he had created for the convection assessment, so this is not for merging into master. 
Perhaps I needed to turn on a flag to indicate this?

Reply:
Thanks! I didn't realize that.